src/redux/categories/categories.js

```diff
@@ -1,19 +1,19 @@
-// ACTION
-const CHECK_STATUS = 'bookstore/catagories/CHECK_STATUS';
+// Categories Reducer
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
 
-// ACTION CREATORS
-export const checkStatus = () => ({
-  type: CHECK_STATUS,
-});
-
-// REDUCER
-const categoryReducer = (state = [], action) => {
+const categories = [];
+const cateReducer = (state = categories, action) => {
   switch (action.type) {
     case CHECK_STATUS:
-      return 'Under Construction';
+      return 'Under construction';
     default:
       return state;
   }
```

Comment:
![check-status (copy)](https://user-images.githubusercontent.com/93039359/186644259-1162032c-1115-4764-84a9-5346e2957537.png)

- [ ] When you click on the Check Status button then a dispatch function should run which should show a string on the page as shown in the green box :+1: